docs/baking.md

```diff
@@ -37,28 +37,29 @@ The most convenient way to orchestrate all these binaries is to use the `tezos-b
 package, which provides predefined services for running baking instances on different
 networks.
 
-To install them, run the following commands:
+#### Add the package repository
+
+On Ubuntu:
 
-<a name="ubuntu"></a>
```

Comment:
This anchor and the anchor below were used in the `Quick Start` section

Reply:
fixed, replaced with link to anchor created from section header